### ISSUE TITLE
Can't get it working in IDE or TTS

### ISSUE THREAD

**Issue by powershellwhizz** [onelivesleft/TTSConsole#1]
HI there,
Unsure if this is still a live project but hope it is! I can't for the life of me figure out what I am doing wrong. I cannot get it this working at all. I came here through downloading a TTS Lua extension in Vscode, that didn't work, so I eventually found your page and tried installing it manually. It still does not work in vscode _and_ doesn't work in TTS either. Here is what I did manually based on your guide.

Downloaded your 2 TTS console files.
Saved them to `<your user folder>/Documents/Tabletop Simulator/Console `as you suggested

I don't have Atom so didn't do anything there, but I assume I should still have access to the console now in TTS at least? I looked at your tutorial which shows you running console command ingame like `ls`. I try this and get an error saying command unknown. Any idea what I am doing wrong here?

PS. My documents folder is in another (custom) location, but to rule this out being a problem, I even duplicated all the files and paths to all different places "documents" could exist. Still no joy!
Hope you can help,
thanks!

**Comment by onelivesleft** [comment]
I'm not sure why you're following the instructions for using it with Atom when you don't have Atom... instead you should do what it says below that: 
```
If you're using a different editor then you can simply paste the code for console.ttslua 
into your own. If you want to use console++.ttslua then paste it below that, and 
remove the #include console it starts with.
```

Open `console.ttslua`, select all, copy, open your code, put cursor at top, paste.

**Comment by powershellwhizz** [comment]
HI,
thanks for your prompt response. Sorry, I did try that as well, just forgot to mention it. I may not have done it 100% correctly then, here is what I did:

Opened the global.lua file from in vscode. copy and pasted the contents from first file, copy and pasted second file and removed the #include part. I tried two ways, I put it inside the onLoad function, and I tried adding it after the function. None worked, I did not put _before_ the function, would that really make a difference? I will try it again at some point today.

Also, can it just go in the global.lua file or does it have to go in all lua files if I want to debug each file?
Thanks again!
